Setting up denoify

garronej opened the pull request:
It should be working 🚀

ovesco commented:
As reflect-metadata is not supposed to be included by the library itself, I modified the generated deno dist files to remove the reflect-metadata import but outside of it works flawlessly thanks! :)

garronej commented:
> As reflect-metadata is not supposed to be included by the library itself, I modified the generated deno dist files to remove the reflect-metadata import but outside of it works flawlessly thanks! :)

Is that so? I am quite confused. If reflect-metadata isn't imported `Reflect.getMetadata` will gives runtime errors won't it? 
If it is not supposed to be included you should just remove the `import  "reflect-metadata";` that I added in the `index.ts` file. If you remove that denoify won't generate an import.
